go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/turbot/steampipe-plugin-slack
 go 1.18
 
 require (
-	github.com/slack-go/slack v0.7.4
+	github.com/slack-go/slack v0.11.3
```

Comment:
I had a look through the releases pages (https://github.com/slack-go/slack/releases and https://github.com/slack-go/slack/releases?page=2), since their changelog isn't up to date, and found a few breaking changes, but none seem to affect our tables, so I think OK to bump here.